src/App.jsx

```diff
@@ -1,14 +1,32 @@
-import React from "react"
+import React, {useEffect, useState} from "react"
 import {Header} from "./components/Header"
+import {TalentCard} from "./components/CardsList/components/TalentCard"
 import {CssBaseline, ThemeProvider} from "@mui/material"
 import {theme} from "./Theme"
+import {MockInit} from "./shared/api/mock/mock"
+import axios from "axios"
 
 const App = () => {
+    const [isLoaded, setIsLoaded] = useState(false)
+    const [talents, setTalents] = useState([])
+
+    MockInit()
+
+    useEffect(() => {
+        axios
+            .get("/talents")
+            .then((response) => {
+                setTalents(response.data)
+                setIsLoaded(true)
+            })
+            .catch(() => {})
```

Comment:
Please use [services](https://github.com/Vladyslav-Team/frontend/tree/5ee84a591a2b505d89e4b13bd169046b6827a962/src/shared/api/services) but  [services should be in component folders](https://github.com/kerrimov/the-ssspace/tree/master/src/components/EventsPage/api/services)  and in app.js must be  https://github.com/Vladyslav-Team/frontend/blob/5ee84a591a2b505d89e4b13bd169046b6827a962/src/App.jsx#L6-L11